library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -465,6 +470,40 @@
 		color: #9E9E9E;
 	}
 
+	.contentReviewDisclosure {
+		position: relative;
+		display: flex;
```

Comment:
Could use the mixins for flex and align-items here, but we'll probably be getting rid of them soon, so not super important.